### ISSUE TITLE
Home - User cannot see menu options when accessibility option is active

### ISSUE THREAD

**Issue by PauloCSto** [topcoder-platform/Training-Challenges#338]
**Steps to Play:**
1. Open URL: www.topcode.com
2. Activate the accessibility option
2. Close the hint banner at the top of the screen
3. Click on the page below the Topcoder logo
4. Press tab to the Community option
5. Click on the "What Can I do" menu

**Current Result:**
The "What Can I do" Menu options do not appear when the user clicks it after using the TAB key

**Expected Result:**
The "What Can I do" Menu options should appear normally when the user clicks on it after using the TAB key

**Lap Top Samsung Expert Line Windows 10 / Firefox 99.0.1**
![Test_Capturar_2022_04_27_21_13_50_971](https://user-images.githubusercontent.com/44380579/165652477-8c5fe934-b8a9-4707-8e6b-6415a0c364c8.jpg)

https://user-images.githubusercontent.com/44380579/165652543-9f9a325d-a5d9-434b-ab4d-088b834b0a43.mp4

**Comment by PauloCSto** [comment]
**Type Bug: UX / Usability Bug**

**Comment by sdgun** [comment]
Accepted
